### PR TITLE
Done the stuff I was specified to do this week:

### PR DESCRIPTION
**Pull request by ValliereMagic**
- Favicon Icon
- Logging
- Deployment notes for TLS

Link for PDF: https://github.com/ValliereMagic/NPLBAM/blob/539e56bfecc96e3af885b466f7c689d08996db69/docs/DeploymentNotes.pdf

### CODE REVIEW

File: nginx/nginx.conf [hunk 1]
```diff
@@ -9,17 +9,21 @@ events {
 }
 # THIS WILL CHANGE AND HAVE TLS SUPPORT (IN PROD)
```
Comment (NevasPrime) [inline nginx/nginx.conf]: Is this comment still needed?

File: nplbam/app/gallery.py [hunk 2]
```diff
@@ -176,18 +180,24 @@ def stage_completed():
                 animal.stage += 1
             elif animal.stage > 8:
                 animal.stage = 8
+            animal_stage: int = animal.stage
```
Comment (NevasPrime) [inline nplbam/app/gallery.py:183]: Needlessly using extra variable

File: nplbam/app/gallery.py [hunk 3]
```diff
@@ -209,8 +218,9 @@ def gallery_info_updated():
         db_session = (sessionmaker(bind=engine))()
 
         # Get our animal from the database
-        animal = db_session.query(db.Animals).filter_by(
+        animal: db.Animals = db_session.query(db.Animals).filter_by(
             animalID=animalID).first()
+        animal_name: str = animal.name
```
Comment (NevasPrime) [inline nplbam/app/gallery.py:223]: Another needless variable

**Review by NevasPrime — CHANGES_REQUESTED** [review]

File: nplbam/app/gallery.py (hunk 2 above)
Comment (NevasPrime) [reply nplbam/app/gallery.py:183]: Apparently I was wrong

File: nplbam/app/gallery.py (hunk 3 above)
Comment (NevasPrime) [reply nplbam/app/gallery.py:223]: Apparently I was wrong. Needed since session is complete

File: nginx/nginx.conf (hunk 1 above)
Comment (ValliereMagic) [reply nginx/nginx.conf]: Nope, it is not. I will remove it.

**Review by NevasPrime — APPROVED** [review]

**Pull request merged**